Traceback (most recent call last):
  File "/global/home/users/kalatimer/proj/20supersat/src/wrf/filtering_criteria_datasrc.py", line 14, in <module>
    from wrf.ss_functions import get_lwc, get_ss_qss, linregress
  File "/global/home/users/kalatimer/proj/20supersat/src/wrf/ss_functions.py", line 51
    def get_ss_pred(ss_qss)
                          ^
SyntaxError: invalid syntax
/global/home/users/kalatimer/proj/20supersat/src/wrf/ss_functions.py:82: RuntimeWarning: invalid value encountered in true_divide
  return meanr/nconc
Traceback (most recent call last):
  File "/global/home/users/kalatimer/proj/20supersat/src/wrf/filtering_criteria_datasrc.py", line 139, in <module>
    main()
  File "/global/home/users/kalatimer/proj/20supersat/src/wrf/filtering_criteria_datasrc.py", line 55, in main
    full_ss, incl_rain, incl_vent)
  File "/global/home/users/kalatimer/proj/20supersat/src/wrf/filtering_criteria_datasrc.py", line 119, in get_filter_dependent_values
    ss_pred_filt = ss_pred[filter_inds]
NameError: name 'ss_pred' is not defined
/global/home/users/kalatimer/proj/20supersat/src/wrf/ss_functions.py:82: RuntimeWarning: invalid value encountered in true_divide
  return meanr/nconc
slurmstepd: error: *** JOB 31371090 ON n0264.lr6 CANCELLED AT 2021-05-18T09:51:27 ***
/global/home/users/kalatimer/proj/20supersat/src/wrf/ss_functions.py:82: RuntimeWarning: invalid value encountered in true_divide
  return meanr/nconc
/global/home/users/kalatimer/proj/20supersat/src/wrf/ss_functions.py:186: RuntimeWarning: invalid value encountered in sqrt
  t = r*np.sqrt(df/((1.0-r+TINY)*(1.0+r+TINY)))
/global/home/users/kalatimer/proj/20supersat/src/wrf/ss_functions.py:187: RuntimeWarning: invalid value encountered in double_scalars
  slope = r_num / ssxm
/global/home/users/kalatimer/proj/20supersat/src/wrf/ss_functions.py:189: RuntimeWarning: invalid value encountered in double_scalars
  sterrest = np.sqrt((1-r*r)*ssym / ssxm / df)
